common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1078,6 +1078,11 @@
          */
         public static final String BROWSER_EXT_WEB_CP = "companyportal://";
 
+        /**
+         * Prefix in the redirect from WebCP.
+         */
+        public static final String AUTHENTICATOR_MFA_LINKING = "microsoft-authenticator://activatemfa";
```

Comment:
>AUTHENTICATOR_MFA_LINKING [](start = 35, length = 25)

I would prefer PREFIX to be added at the end of the constant name AUTHENTICATOR_MFA_LINKING_PREFIX,